test/run_test.py

```diff
@@ -401,8 +401,8 @@
 SPECIFIED_TEST_CASES_DICT: Dict[str, List[str]] = {}
 
 # The file from which the SPECIFIED_TEST_CASES_DICT will be filled, a CSV of test cases that would be run when
-# options.run_specified_test_cases is enabled.
-SPECIFIED_TEST_CASES_FILE: str = '.pytorch_specified_test_cases.csv'
+# options.use_specified_test_cases is enabled.
+SPECIFIED_TEST_CASES_FILE: str = '.pytorch_use_specified_test_cases.csv'
```

Comment:
Why do we want to change the name of the csv?

Reply:
oops........ regex search/replace error. undoing